### DISCUSSION TITLE
有考虑出海面向非中文的用户么

### DISCUSSION THREAD

**Discussion by mzhang001** [Kenshin/simpread#6306] [讨论组]
我在国外工作了很多年，感觉你的插件做的还蛮好的，只不过好像目前还主要服务于中文用户。

有考虑面过国际非中文用户么？有什么我能帮忙的么，谢谢～

**Comment by Kenshin** [comment]
谢谢习惯并支持简悦。

之前用户也提过类似的建议，只是「英文化」所需的文案太多了，而且国内与国外的网站结构差别也挺大的。

当然，出海是肯定要做的事情。

请问你有这方面的建议吗？
